src/event.js

```diff
@@ -0,0 +1,41 @@
+export class Event {
```

Comment:
`window.Event` is already a reserved global class, so I'd recommend using something else to avoid confusion and collisions.